paddle/fluid/inference/api/analysis_predictor.cc

```diff
@@ -378,6 +377,17 @@ void AnalysisPredictor::OptimizeInferenceProgram() {
     argument_.SetMKLDNNEnabledOpTypes(config_.mkldnn_enabled_op_types_);
   }
 
+  if (config_.quantizer_enabled()) {
+    LOG(INFO) << "Quantization is enabled";
+    // TODO(wojtuss): enable below after merge of PR #16326
```

Comment:
@wojtuss #16326 is merged already.

Reply:
fixed in https://github.com/PaddlePaddle/Paddle/pull/16396